src/main/java/com/tagtraum/perf/gcviewer/view/ModelChartImpl.java

```diff
@@ -186,7 +186,7 @@ private void maybePopup(MouseEvent e) {
         addMouseWheelListener(new MouseWheelListener() {
             @Override
             public void mouseWheelMoved(MouseWheelEvent e) {
-                if ((e.getModifiersEx() & InputEvent.SHIFT_DOWN_MASK) != 0) {
+                if ((e.getModifiersEx() & InputEvent.META_DOWN_MASK) != 0||(e.getModifiersEx() & InputEvent.CTRL_DOWN_MASK) != 0) {
```

Comment:
I would rather make the modifier OS-dependent.
In other words, `META` for macOS, and `CTRL` for the rest.

Just in case, ctrl+click means "right click" in macOS

Reply:
I have pushed a commit to adding the os dependent code using OSXSupport.isOSX().